### DISCUSSION TITLE
"Dynamic Image Generation" enabled on WP Engine

### DISCUSSION THREAD

**Discussion by obritoluis** [flyntwp/flynt#360] [Q&A] [answered]
Have you ever got the "Dynamic Image Generation" working on WP Engine? They mix apache with nginx and following this [guide](https://github.com/flyntwp/flynt#troubleshooting-1) we asked them to add this on the nginx config:

`location ~ "^(.*)/wp-content/uploads/(.*)$" {
  try_files $uri $uri/ /index.php$is_args$args;
}`

But it didn't work.

Wondering if you managed to make it work. Thanks 🙂

**Comment by filipe-q** [comment]
I also have the same issue and still couldn't get it fixed. Would definitely appreciate the input from someone in the Flynt development team

**Comment by domtra** [comment, marked as answer]
Hi @obritoluis and @filipe-q ,
we managed to get it to work on a project, after two long chats with their support team.
The rule that got dynamic image generation to work was

```
location ~* /wp-content/uploads/dynamic/ { proxy_pass $dynamic_upstream_read_write; }
```

If you use a newer version of flynt, it should be
```
location ~* /wp-content/uploads/resized/ { proxy_pass $dynamic_upstream_read_write; }
```

Let me know if that helps.

**Reply by obritoluis** [reply]
Thanks @domtra ! I'll try 🙂
